cmd/proto-schema-registry/main.go

```diff
@@ -0,0 +1,81 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"net"
+	"os"
+	"os/signal"
+	"syscall"
+
+	"github.com/syncromatics/proto-schema-registry/internal/log"
+	"github.com/syncromatics/proto-schema-registry/internal/service"
+
+	"github.com/pkg/errors"
+	"golang.org/x/sync/errgroup"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/reflection"
+)
+
+func main() {
+	settings, err := getSettingsFromEnv()
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	log.Info(settings.String())
+
+	server := grpc.NewServer()
```

Comment:
Thoughts on passing in an optional flag to disable TLS here? At least I think that's the right spot:
https://github.com/grpc/grpc-go/blob/d8e3da36ac481ef00e510ca119f6b68177713689/server.go#L393

Reply:
Whoops, it *is* off by default, your IRL comment was right
https://github.com/grpc/grpc-go/blob/master/interop/server/server.go#L55